cdap-docs/developers-manual/source/building-blocks/datasets/index.rst

```diff
@@ -178,6 +177,11 @@ such as the format and schema, while abstracting from the actual underlying file
 - An extension of FileSets, ``PartitionedFileSets`` allow the associating of meta data (partitioning keys)
```

Comment:
extension is - strictly speaking - incorrect as it suggests that it is a subclass (yes we are in Java land). More correctly, it is an encapsulation, but that sounds odd here. Can we say "abstraction"?